packages/driver/src/cypress/log.ts

```diff
@@ -505,8 +505,11 @@ class Log {
 
       consoleObj[key] = _this.get('name')
 
+      // in the case a log is being recreated from the cross-origin spec bridge to the primary, consoleProps may be an Object
+      const consoleObjDefaults = _.isFunction(consoleProps) ? consoleProps.apply(this, args) : consoleProps
+
       // merge in the other properties from consoleProps
-      _.extend(consoleObj, consoleProps.apply(this, args))
+      _.extend(consoleObj, consoleObjDefaults)
```

Comment:
FYI: We still support the deprecated log attribute of `onConsole` that users might pull in. Does the origin logs handle this?

https://github.com/cypress-io/cypress/blob/md-support-snapshots-and-console-props/packages/driver/src/cypress/log.ts#L302

Reply:
I'm not too familiar with `onConsole`, but it should still work since `onConsole` gets converted to `consoleProps` in the spec bridge, and then `consoleProps` is ultimately serialized back to the primary. Do you know where I can find more information on `onConsole`? Want to make sure this is the case 😅